HtmlCssReview/README.md

```diff
@@ -0,0 +1,37 @@
+# HTML & CSS Review
+In this lesson we will review content from HTML and CSS.
+
+## Icebreaker
+🤖 Note: you will need an OpenAI account.
+
+The [GPT-3 Mad Libs activity](https://platform.openai.com/playground/p/JQHsFLQt2yZuMc6VZfGDdR7K?model=text-davinci-003) is great for the first day because it allows the instructors to get familiar with saying and typing the students' names.
+
+1. Go around the room and have each student say their **name** and a **fact** about themself, and type that into the top half of the prompt (replacing the sample list)
+
+1. Go around again and, saying each student's name out loud, ask them to provide **a word** for the bottom half (any remaining category they want)
+
+1. Adjust the temperature, submit, and read the results out loud for maximum enjoyment 😄
+
+If there are any issues, fall back to [this Mad Libs Icebreaker](https://github.com/hylandtechoutreach/ucs/blob/main/Icebreakers/MadLibsIcebreaker.md).
+
+## Code-Along: HTML & CSS Review
+💡 Note: the students should already be familiar with Replit, but if they need a refresher, direct them [here](https://hylandtechclub.com/ReplitSetup).
+
+Walk through [the code-along instructions](HtmlCssReviewCodeAlong.md) with the students to create a Fortnite fan web site. Go slowly to be sure everyone is following along.
+
+If you end up with time left over, have the students change the site to be about something else.
+
+## Kahoot
+Run through this [Kahoot](https://create.kahoot.it/share/9da0c22b-48ae-479f-9c7f-c0bfd458a2b5) to gauge what the students ultimately remember from HTML/CSS.
```

Comment:
Kahoot was being slow for me so I couldn't check this quiz - but we will want to make sure it lines up with everything from the code-along

Reply:
Resolved in the latest commit.